lte/gateway/fabfile.py

```diff
@@ -250,7 +250,8 @@ def federated_integ_test(
 
 def integ_test(
     gateway_host=None, test_host=None, trf_host=None,
-    destroy_vm='True', provision_vm='True',
+    destroy_vm='True', build=None, hash=None,
+    provision_vm='True',
```

Comment:
can we just use `hash` or `github_hash`?

If hash is empty, then you build it from source. If we have a hash, then pull it from repo

```
def integ_test(
    gateway_host=None, test_host=None, trf_host=None,
    destroy_vm='True', github_hash='',
    provision_vm='True',
```